File: frontend/src/api/feedback.js
import client from './client';

export const createFeedback = (data) => client.post('/feedbacks', data);
export const getFeedbacks = (params) => client.get('/feedbacks', { params });
export const getFeedbackDetail = (id) => client.get(`/feedbacks/${id}`);
export const updateFeedback = (id, data) => client.put(`/feedbacks/${id}`, data);

File: backend/src/routes/feedback.rs
use axum::{routing::{get, post, put}, Router};
use sqlx::MySqlPool;
use crate::handlers::feedback::{
    create_feedback, get_feedback_detail, get_feedbacks, update_feedback,
};

pub fn feedback_router() -> Router<MySqlPool> {
    Router::new()
        .route("/feedbacks",     post(create_feedback).get(get_feedbacks))
        .route("/feedbacks/:id", get(get_feedback_detail).put(update_feedback))
}
